webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/ShowNodeDetailsFromNode.scala

```diff
@@ -87,38 +90,39 @@ class ShowNodeDetailsFromNode(
 ) extends DispatchSnippet with DefaultExtendableSnippet[ShowNodeDetailsFromNode] with Loggable {
   import ShowNodeDetailsFromNode._
 
-  private[this] val nodeInfoService      = RudderConfig.nodeInfoService
   private[this] val nodeFactRepo         = RudderConfig.nodeFactRepository
   private[this] val reportDisplayer      = RudderConfig.reportDisplayer
   private[this] val logDisplayer         = RudderConfig.logDisplayer
   private[this] val uuidGen              = RudderConfig.stringUuidGenerator
-  private[this] val nodeRepo             = RudderConfig.woNodeRepository
   private[this] val asyncDeploymentAgent = RudderConfig.asyncDeploymentAgent
   private[this] val configService        = RudderConfig.configService
-  private[this] val boxNodeInfo          = nodeInfoService.getNodeInfo(nodeId).toBox
 
   def agentPolicyModeEditForm = new AgentPolicyModeEditForm()
 
```

Comment:
Here (), the removal of `nodeInfoService` lead to substantial, non interesting changes and some simplification since we have `NodeFact` around, not just sometime inventory and sometime node info.